test/helpers/cilium.go

```diff
@@ -632,6 +633,34 @@ func (s *SSHMeta) ReportFailed(commands ...string) {
 	s.GatherDockerLogs()
 }
 
+// ValidateEndpointsAreCorrect is a function that validates that all Docker
+// container that are in the given docker network are correct as cilium
```

Comment:
Instead of 'correct', I would say 'that all containers in the Docker network have a corresponding endpoint in cilium'.